app/controllers/Application.scala

```diff
@@ -102,4 +102,9 @@ class Application(
   def healthcheck: Action[AnyContent] = PrivateAction {
     Ok("healthy")
   }
+
+  // Remove trailing slashes so that /uk/ redirects to /uk
+  def removeTrailingSlash(path: String): Action[AnyContent] = CachedAction() {
+    MovedPermanently("/" + path)
+  }
```

Comment:
I don't think query params are an issue in this case it's more to catch people typing in a url incorrectly. I can't imagine anyone would type `/uk/?blah=thing` can you?

Reply:
It would be nice to keep the query params just in case someone setup a link with tracking data incorrectly (i.e. with the final `/`). In that case we will be losing all the tracking information.

Reply:
Done.